README.md

```diff
@@ -117,4 +117,214 @@ containers:
+
+```
+apiVersion: dbaas.redhat.com/v1alpha1
+kind: DBaaSProvider
+metadata:
+  labels:
+    related-to: dbaas-operator
+    type: dbaas-provider-registration
+  name: mongodb-atlas-registration
+spec:
+  connectionKind: MongoDBAtlasConnection
+  credentialFields:
+  - displayName: Organization ID
+    key: orgId
+    required: true
+    type: string
+  - displayName: Public API Key
+    key: publicApiKey
+    required: true
+    type: string
+  - displayName: Private API Key
+    key: privateApiKey
+    required: true
+    type: maskedstring
+  inventoryKind: MongoDBAtlasInventory
+  provider:
+    displayDescription: Cloud-hosted MongoDB service on AWS, Azure and Google Cloud
+    displayName: MongoDB Atlas Cloud Database Service
+    icon:
+      base64data: <icon image data>
+      mediatype: image/png
+    name: Red Hat DBaaS / MongoDB Atlas
+```
+If the Atlas Operator is undeployed with the OLM, the above registration CR gets cleaned up automatically.
+
+**2.** Check MongoDBAtlasInventory Customer Resource
+
+When an application developer creates a [DBaaSInventory](https://github.com/RHEcosystemAppEng/dbaas-operator/blob/main/config/crd/bases/dbaas.redhat.com_dbaasinventories.yaml) CR for MongoDB, the DBaaS Operator creates a MongoDBAtlasInventory CR, and the Atlas Operator discovers the clusters and  instances, and sets the result in the CR status. 
+Here is an example of MongoDBAtlasInventory CR.
+```
+apiVersion: dbaas.redhat.com/v1alpha1
+kind: MongoDBAtlasInventory
+metadata:
+  name: dbaas-mytest
+  namespace: openshift-operators
+  ownerReferences:
+  - apiVersion: dbaas.redhat.com/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: DBaaSInventory
+    name: dbaas-mytest
+    uid: 01f5a690-c640-462f-b6e8-ccb9db95df70
+  resourceVersion: "950335526"
+  selfLink: /apis/dbaas.redhat.com/v1alpha1/namespaces/openshift-operators/mongodbatlasinventories/dbaas-mytest
+  uid: cec65b27-7192-4e13-bfe7-efb6135f3c7c
+spec:
+  credentialsRef:
+    name: my-atlas-key
+    namespace: openshift-operators
+status:
+  conditions:
+  - lastTransitionTime: "2021-08-18T20:06:47Z"
+    message: Spec sync OK
+    reason: SyncOK
+    status: "True"
+    type: SpecSynced
+  instances:
+  - instanceID: 608df625aa94426b4167e45f
+    instanceInfo:
+      connectionStringsStandard: ""
+      connectionStringsStandardSrv: mongodb+srv://dbaas-cluster1.h38mo.mongodb.net
+      instanceSizeName: M0
+      projectID: 608df5e652e1944293e8584a
+      projectName: Project 1
+      providerName: TENANT
+      regionName: US_EAST_1
+    name: DBaaS-Cluster1
+  - instanceID: 60807282b4ab8d3b3c84be53
+    instanceInfo:
+      connectionStringsStandard: mongodb://test-shard-00-00.iojsa.mongodb.net:27017,test-shard-00-01.iojsa.mongodb.net:27017,test-shard-00-02.iojsa.mongodb.net:27017/?ssl=true&authSource=admin&replicaSet=atlas-133pql-shard-0
```

Comment:
are not change this to show as a host  and isSrv field ?, also not sure we need to show the connection details here

Reply:
Removed connectionStringsStandard as it is not used currently. connectionStringsStandardSrv is kept in the inventory as it makes a little issue for troubleshooting.